scripts/xtensa-build-all.sh

```diff
@@ -330,12 +321,12 @@ do
 			HOST="xtensa-rn-elf"
 			XTENSA_TOOLS_VERSION="RF-2016.4-linux"
 			;;
-                mt8186)
-                        PLATFORM="mt8186"
-                        XTENSA_CORE="hifi5_7stg_I64D128"
-                        HOST="xtensa-mt8186-elf"
-                        XTENSA_TOOLS_VERSION="RI-2020.5-linux"
-                        ;;
+		mt8186)
+			PLATFORM="mt8186"
+			XTENSA_CORE="hifi5_7stg_I64D128"
+			HOST="xtensa-mt8186-elf"
+			XTENSA_TOOLS_VERSION="RI-2020.5-linux"
+			;;
 		mt8195)
```

Comment:
what is this change here? It doesn't say anything about it in the commit message? If this is indeed a change, can you please add it in a separate commit and explained what happened. I cannot figure it out from github diff. Maybe you changed some tabs? :)

Reply:
This is the message of the second commit:
> [xtensa-build-all.sh: Code style correction](https://github.com/thesofproject/sof/pull/5475/commits/0affed20e9d99ced951a2fe31c79521ebfa75135) 
>
> Combined case label tgl and tgl-h similarly as in xtensa-build-zephyr.sh
> Corrected code formatting.

@dbaluta the Github description / "cover letter" and the git commit messages are different.